src/reanimated2/layoutReanimation/web/animationsManager.ts

```diff
@@ -115,6 +115,12 @@ export function startWebLayoutAnimation<
   animationType: LayoutAnimationType,
   transitionData?: TransitionData
 ) {
+  if (element.style.transform !== '') {
+    console.warn(
+      `[Reanimated] Property "transform" will not be included in layout animation. Please wrap your component with an animated view and apply the layout animation on the wrapper.`
```

Comment:
I am not sure whether every user will understand that `apply the layout animation on the wrapper` means `move the layout animation to the wrapper`. I'd make it a little bit more straightforward stating that it has to be moved there.